.github/workflows/go-crypto.yml

```diff
@@ -0,0 +1,42 @@
+# Workflow to test different crypto backends
+# Don't test std go as this is already tested by the other testsuite
+
+on: [push, pull_request]
+name: Go Crypto
+
+jobs:
+  unit:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "ubuntu" ]
+        go: [ "1.19.x" ]
+        backend: [ "boringcrypto", "openssl" ]
+    env:
+      COVERAGES: ""
+    runs-on: ${{ format('{0}-latest', matrix.os) }}
+    name: ${{ matrix.os }} ${{ matrix.backend }} (go ${{ matrix.go }})
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+      - name: Go information
+        run: |
+          go version
+          go env
+      - name: Run repo-specific setup
+        uses: ./.github/actions/go-test-setup
+        if: hashFiles('./.github/actions/go-test-setup') != ''
+      - name: Run tests
+        run: |
+          if [[ "$backend" == "boringcrypto" ]]; then export GOEXPERIMENT="boringcrypto"; fi
```

Comment:
Oh, you can use the standard toolchain you just have to set this flag. I thought you had to build the Go compiler. :+1:

Reply:
Same, this also work but you dont actually need it.